src/Aptos/MoveTypes.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright © 2017-2022 Trust Wallet.
+// Copyright © 2017-2023 Trust Wallet.
```

Comment:
Maybe next time it's better to submit this kind of changes as a separate PR, cause it's really hard to find what to review here

Reply:
You can review commit by commit in the review

Reply:
<img width="391" alt="Screenshot 2023-03-14 at 14 45 33" src="https://user-images.githubusercontent.com/21139416/225020980-6a3bc993-1de0-4e85-80b1-aea675854429.png">